scripts/helpers.js

```diff
@@ -1,16 +1,40 @@
+var moment = require('moment');
 var urllib = require('url');
 
 var utils = require('../lib/utils');
-
 var pkg = require('../package');
+
+var isUrl = utils.isUrl;
+var urljoin = utils.urljoin;
+
 var aframeCurrentSha = 'master';
 try {
   aframeCurrentSha = pkg.dependencies.aframe.split('#')[1];
-} catch (e) {
-}
+} catch (e) {}
 
-var isUrl = utils.isUrl;
-var urljoin = utils.urljoin;
+hexo.extend.helper.register('blog_attribution', function (author) {
+  // Twitter handle.
```

Comment:
it'll probably just complicate things, but we might want to strip a leading `@`:

```js
author = author.split('@')[1];
```